### ISSUE TITLE
Totally wrong prediction during testing on pretrained model

### ISSUE THREAD

**Issue by ChenyuLInx** [SeanNaren/CTCNetworkClient#92]
Hi there,

Thanks for the great implementation.
I tried to use the LibriSpeech pre-trained model you provided on the website. When I test it with my own data. I always got the wrong prediction. Any idea why the performance is so off?

Here's my result:

> smssysysysdysmsmsysmsdsdsdysydsssskskpsbsmsbysyssbslbsbbysllssmsbdsgsdskssdsdsdsbsdsysysdsmlsmsdsyskysydsydyybymsbysssmsysmsmsysdbsdsdsyysssysmsbsbysysbmsmsbmbsbsmsysysmsbsmsmsbsbsysdksdlsmrusmsdssypsksdsbsdsdsydsysxspsmsmsmsysmbsysmsbybkdysksdysmssdsysmsssbssnyskskspssmsdypbyssdlskysmsdsysymsmsmspsdsysbsmsmsysmsdydsxsbsmpmssmsysdysysmysmsxsdsyyysbdsksdssysmsmsxsmsdsssmsmsmsmsrssspsdsbssmbsmsssdydsymysbmsmsdysbsbssmsdssddssddsdsdsdsdssspsddysdssspmbspsydsbsispsyysdsysyspsysysysdsysddpdbyssdssbsywssdsdpsbmsdspdsdssssmsmsmsmsbsmssmsmsbsyssdsbsdsdsdkdsddsdydbddksymbsbsdsxspssmspsylsbsmsyscssmsmsdysbsysysydssbsksmsybsmsmsxssmssysmsdmpsmspsdsddsdssdksmsdsbdsdgysgkpsmsmsmsdbsdsdydssdysmsmsbssybssmsmslsmsnxmsmbpsbsdyklsmpsmsdsdsssysyssbsbsbspsdsmsmsmspsdsdydgdsydkdsdsssbsbsbsbsdsbmbbsspsmsmbsmpbsdysyspsbsddyssysuspssysbsbsyspsdsdssmsmsmsybsksysyksxkyssysysydybsdpssdssysmsmsmymsmsbsdsdysysksdsdssdsmsdylslbdybdsykfsmssmsmsmsmsmsysksskskdsskssssdysssdsyskydsdyssdddsdsdsysysbsdsdksdsdslsysysysmsmsmspsmsdsysysmsmsmsmsdysmsmssbsysdysdssusmsmsmsmsmsmsdspslslspsdsbyssybsspbsdssdmsdsmsbsysdsdssmsbsbspsdisbsbssmsbsdsdssymsbsmsmdspsdsssmsbspysbsdysbsyssmsmrumsdpsysysyssysysysdsdyyssssmbspsisbskssmsysydydsdbdbsbbd

Thank you so much.
Chenyu

**Comment by ArtemiyFirsov** [comment]
Yes, same for me (
sssmsymsmsbsbsbsblmsmsbsdbysssdmlsdssrsmsmsmsysmsmspsysmsmsmsmsmsysdsdsydsmsmsmsmsmsmsmsmsmsdsdsmsmsdymsdysmsmsmsmsysmsmsmsmsmsmsmpsmsmsdysysbsdssdssg   

Maybe it is underfitting problem? 
Converted audio to .wav 16kHz with 1 channel an tried 8bit and 16bit. Same everywhere.

**Comment by shantanudev** [comment]
@SeanNaren Yes, I am also facing the same exact issue since I have had to update my system. Is this a problem with the CTC function?
